jaseci_core/jaseci/api/config_api.py

```diff
@@ -47,25 +52,61 @@ def config_set(self, name: str, value: str or dict, do_check: bool = True):
         self._h.save_glob(name, value)
         return [f"Config of '{name}' to '{value}' set!"]
 
+    @Interface.admin_api()
+    def config_yaml(self, name: str, file: list):
+        """
+        Set a config from yaml
+        """
+
+        new_config = {}
+
+        config_version = str(time())
+
+        for conf in yaml.safe_load_all(b64decode(file[0]["base64"])):
+            kind = conf["kind"]
+            labels = conf.get("metadata").get("labels")
+            if not labels.get("config_version"):
+                labels["config_version"] = config_version
+
+            if not new_config.get(kind):
+                new_config[kind] = []
+            new_config[kind].append(conf)
+
+        old_config = self._h.get_glob(name)
+        if old_config:
+            old_config = loads(old_config)
+            old_config.pop("__OLD_CONFIG__", None)
+            for kind, confs in old_config.items():
+
+                print(kind)
+                names = []
+                for conf in confs:
+                    names.append(conf["metadata"]["name"])
+                old_config[kind] = names
+
+            new_config["__OLD_CONFIG__"] = old_config
+
+        self._h.save_glob(name, dumps(new_config))
+
+        return new_config
+
     @Interface.admin_api()
     def config_refresh(self, name: str):
         """
-        update global configs
+        refresh service configs
         """
 
         hook = self._h
-        if name == "TASK_CONFIG":
-            hook.task.reset(hook)
-        elif name == "REDIS_CONFIG":
-            hook.redis.reset(hook)
-        elif name == "MAIL_CONFIG":
-            hook.mail.reset(hook)
-        elif name == "PROMON_CONFIG":
-            hook.promon.reset(hook)
-        elif name == "KUBE_CONFIG":
-            hook.kube.reset(hook)
-        elif name == "JSORC_CONFIG":
-            hook.jsorc.reset(hook)
+
+        to_start = not hook.meta.is_automated()
+
+        service = getattr(hook, name, None)
+
+        if isinstance(service, CommonService):
+            service.reset(hook, to_start)
+            return
+
+        return "Not a valid operation!"
```

Comment:
I think we need to return a JSON for APIs always so let's make these two returns all JSON and following our convention of
```
{
  "success": true/false,
}
```
And for the line 109, i'd add the message to the response and I'd make it more accurate to something like
```
f"{name} is not a valid service. Can not refresh config."
```